src/tests/codegen-backend-tests/cg_gcc.md

```diff
@@ -1,3 +1,34 @@
 # GCC codegen backend tests
 
-TODO: please add some more information to this page.
+To test the GCC codegen backend, you need to add `"gcc"` into the `rust.codegen-backends`
+setting in `bootstrap.toml`:
+
+```toml
+rust.codegen-backends = ["llvm", "gcc"]
+```
+
+If you don't want to change your `bootstrap.toml` file, you can alternatively run your `x.py`
+commands with `--set rust.codegen-backends=["llvm", "gcc"]'`. For example:
+
+```bash
+x.py test --set 'rust.codegen-backends=["llvm", "gcc"]'
+```
+
+If you don't want to build `gcc` yourself, you also need to set:
+
+```toml
+gcc.download-ci-gcc = true
```

Comment:
Do I also have to add this to `--set` to avoid waiting hours until GCC is built?

Why is `true` not the default?